.cargo/config.toml

```diff
@@ -41,3 +41,6 @@ rustflags = [
 	"-Dclippy::unwrap_used",
 	"-Dclippy::expect_used",
 ]
+
+[net]
+git-fetch-with-cli = true
```

Comment:
the random network issues on ubicloud warrants this check 

https://doc.rust-lang.org/cargo/reference/config.html#netgit-fetch-with-cli